setup.cfg

```diff
@@ -0,0 +1,21 @@
+[bumpversion]
+current_version = 0.9.0.0
```

Comment:
Agreed, btw current ucscsdk version is 0.9.0.0 and we plan to release next version of ucscsdk only after ucsc_apis first release. So no changes needed here.